src/NorthwindWebsite/Business/Services/Interfaces/IImageCachingService.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace NorthwindWebsite.Business.Services.Interfaces;
+
+public interface IImageCachingService
+{
+    bool IsContained(string imageIndex);
+
+    int GetNumberOfFilesInCachingFolder();
+
+    void CreateFolderIfDoesNotExists();
```

Comment:
Well, let's split it into 2 separate methods to follow single responsibility principle